configure.ac

```diff
@@ -1062,99 +1062,111 @@ model=default
-    [arch=amd64; system=cygwin],
-  [riscv64-*-linux*],
-    [arch=riscv; model=riscv64; system=linux]
+    [arch=amd64; system=mingw64]
 )
 
-AS_IF([test x"$enable_native_compiler" = "xno"],
-  [native_compiler=false
-  AC_MSG_NOTICE([the native compiler is disabled])],
-  [native_compiler=true])
+# The following platforms support native-code compilation in OCaml 4
+# but have not yet been ported to OCaml 5.
+
+#  [[i[3456]86-*-linux*]],
+#    [arch=i386; system=linux_elf],
+#  [[i[3456]86-*-*bsd*]],
+#    [arch=i386; system=bsd_elf],
+#  [[i[3456]86-*-haiku*]],
+#    [arch=i386; system=beos],
+#  [[i[3456]86-*-cygwin]],
+#    [arch=i386; system=cygwin],
+#  [[i[3456]86-*-gnu*]],
+#    [arch=i386; system=gnu],
+#  [[i[3456]86-*-mingw32]],
+#    [arch=i386; system=mingw],
+#  [i686-pc-windows],
+#    [arch=i386; system=win32],
+#  [x86_64-pc-windows],
+#    [arch=amd64; system=win64],
+#  [[powerpc64le*-*-linux*]],
+#    [arch=power; model=ppc64le; system=elf],
+#  [[powerpc*-*-linux*]],
+#    [arch=power; AS_IF([$arch64],[model=ppc64],[model=ppc]); system=elf],
+#  [[s390x*-*-linux*]],
+#    [arch=s390x; model=z10; system=elf],
+#  # expected to match "gnueabihf" as well as "musleabihf"
+#  [armv6*-*-linux-*eabihf],
+#    [arch=arm; model=armv6; system=linux_eabihf],
+#  [armv7*-*-linux-*eabihf],
+#    [arch=arm; model=armv7; system=linux_eabihf],
+#  [armv8*-*-linux-*eabihf],
+#    [arch=arm; model=armv8; system=linux_eabihf],
+#  [armv8*-*-linux-*eabi],
+#    [arch=arm; model=armv8; system=linux_eabi],
+#  [armv7*-*-linux-*eabi],
+#    [arch=arm; model=armv7; system=linux_eabi],
+#  [armv6t2*-*-linux-*eabi],
+#    [arch=arm; model=armv6t2; system=linux_eabi],
+#  [armv6*-*-linux-*eabi],
+#    [arch=arm; model=armv6; system=linux_eabi],
+#  [armv6*-*-freebsd*],
+#    [arch=arm; model=armv6; system=freebsd],
+#  [earmv6*-*-netbsd*],
+#    [arch=arm; model=armv6; system=netbsd],
+#  [earmv7*-*-netbsd*],
+#    [arch=arm; model=armv7; system=netbsd],
+#  [armv5te*-*-linux-*eabi],
+#    [arch=arm; model=armv5te; system=linux_eabi],
+#  [armv5*-*-linux-*eabi],
+#    [arch=arm; model=armv5; system=linux_eabi],
+#  [arm*-*-linux-*eabihf],
+#    [arch=arm; system=linux_eabihf],
+#  [arm*-*-linux-*eabi],
+#    [arch=arm; system=linux_eabi],
+#  [arm*-*-openbsd*],
+#    [arch=arm; system=bsd],
+#  [zaurus*-*-openbsd*],
+#    [arch=arm; system=bsd],
+#  [x86_64-*-gnu*],
+#    [arch=amd64; system=gnu],
+#  [x86_64-*-dragonfly*],
+#    [arch=amd64; system=dragonfly],
+#  [x86_64-*-solaris*],
+#    [arch=amd64; system=solaris],
+#  [x86_64-*-haiku*],
+#    [arch=amd64; system=beos],
+#  [arm64-*-darwin*],
+#    [arch=arm64; system=macosx],
+#  [aarch64-*-darwin*],
+#    [arch=arm64; system=macosx],
+#  [aarch64-*-linux*],
+#    [arch=arm64; system=linux],
+#  [aarch64-*-freebsd*],
+#    [arch=arm64; system=freebsd],
+#  [x86_64-*-cygwin*],
+#    [arch=amd64; system=cygwin],
+#  [riscv64-*-linux*],
+#    [arch=riscv; model=riscv64; system=linux]
```

Comment:
I think it would be nice to keep these comments positioned in such a way that, if I want to hack on a backend for one of those systems, I can just uncomment the two lines and start running.

Reply:
These are shell comments, not m4 comments, so there's a risk of messing with the m4 macros.

Reply:
I agree that it would be nice to keep the diff quieter - the "appropriate" syntax in this case is to use `dnl` which is m4's comment indicator. I double-checked this in https://github.com/dra27/ocaml/commit/43020f7ab9d4fe69051c79bbed23f0455601b561. By putting `mingw-w64` last, it does mean that deleting the `dnl MVP:` is sufficient to re-add any case, as @gasche requests.